packages/sources/ion.au/src/transport/total_reserve.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BaseEndpointTypes } from '../endpoint/total_reserve'
+
+export interface ResponseSchema {
+  total_reserve: number
+  errorMessage?: string
+}
+
+export type HttpTransportTypes = BaseEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return {
+      params,
+      request: {
+        baseURL: config.API_ENDPOINT,
+        url: '/',
+        headers: {},
+        params: {},
+      },
+    }
+  },
+  parseResponse: ({}, response) => {
```

Comment:
Ah I missed this when reviewing. Could you change this back to `params`
```suggestion
  parseResponse: (params, response) => {
```

Reply:
I cant do this because I get a build error. `params` has to be used somewhere or the program does not build.

Reply:
In that case could you replace it with `_`? I think that'll skip the error

Reply:
This also throws a type error.

`error TS2322: Type '(_: ({} & {})[] ... `

Reply:
Huh interesting. In that case maybe we can just use it. Could you replace all of the `params: {},` in the `parseResponse` method with `params: params[0],`

Reply:
`_params` works well.

Reply:
Sure, do you prefer the `_params` appraoch or replace `{}` with `params[0]` ?